src/pages/api/getDateInfo.ts

```diff
@@ -0,0 +1,16 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import { getDateInfo } from "../../lib/backend/database-utils";
+
+const handler = async (req, res) => {
+    console.log(" in getDateInfo");
+    var data;
+    if (req.method == "POST") {
```

Comment:
This can be simplified by using the next-connect (nc) package that is installed.  Look at some of the other apis for examples.